npm/ng-packs/packages/core/src/lib/core.module.ts

```diff
@@ -5,6 +5,7 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterModule } from '@angular/router';
 import { AbstractNgModelComponent } from './abstracts/ng-model.component';
 import { DynamicLayoutComponent } from './components/dynamic-layout.component';
+import { InternetConnectionStatusComponent } from '../../../theme-shared/src/lib/components/internet-connection-status/internet-connection-status.component';
```

Comment:
export `InternetConnectionStatusComponent ` from `components\index.ts`, so we can import it here shortly